auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/response/ProductPaginated.java

```diff
@@ -0,0 +1,51 @@
+package com.atlantbh.auctionappbackend.response;
+
+import com.atlantbh.auctionappbackend.domain.Product;
+
+import java.util.List;
+
+public class ProductPaginated {
```

Comment:
This class should be generic, so we can use it for all models (not only for products).
```
public class PaginatedResponse<T> {
     private List<T> data;
     ....
}
```